### ISSUE TITLE
Feature Request: Allow sync to a generic folder

### ISSUE THREAD

**Issue by rollercoaster** [minhnhattonthat/ryujinx-tool#3]
First off, great tool!

For purposes of syncing files across systems/backups, can we have an option that can sync to a generic folder using the TitleID format? This would really help with tools like Syncthing!

Thank you.

**Comment by minhnhattonthat** [comment]
hi @rollercoaster, thank you for using the tool!

For your use case, have you considerred using symlink ([mklink](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/mklink) command on WIndows)? It's like generate shortcut for your save folder on other directory location. I also use Syncthing and it really helps.

The reason I want to have user provide the ryujinx root path because to sync save, it actually has to modify another file that is not in the save folder (imkvdb.arc), so this would streamline the process a bit.

Else if there are more interest on this, maybe I can consider adding this in the future.

Thanks,

**Comment by rollercoaster** [comment]
Yes I have aware of that but this tool make it a lot more dynamic. I think the only change would be replacing the yuzu profile path with a regular folder path and continuing the sync unless I am missing something.

Your efforts are much appreciated!
